core-plugin/src/com/google/gct/idea/debugger/CloudDebuggerClient.java

```diff
@@ -105,15 +109,20 @@ public void statusChanged(boolean login) {
           HttpRequestInitializer initializer = new HttpRequestInitializer() {
             @Override
             public void initialize(HttpRequest httpRequest) throws IOException {
+              HttpHeaders headers = new HttpHeaders();
+              headers.setUserAgent(USER_AGENT);
```

Comment:
We want this for all server-side actions including cloud repo syncing and eventually deployment.